custom/icds_reports/static/js/icds_app_mobile.js

```diff
@@ -47,7 +47,8 @@ MainMobileController.$inject = [
     'userLocationId',
 ];
 
-window.angular.module('icdsApp', ['ngRoute', 'cgBusy'])
+// ui.bootstrap not truly needed - but location directive depends on it to compile
+window.angular.module('icdsApp', ['ngRoute', 'cgBusy', 'ui.bootstrap'])
```

Comment:
this is annoying, but I'm not going to bother decoupling the modal UI logic used on web from the `location-filter.directive` at this time.